test/fastify-instance.test.js

```diff
@@ -97,3 +97,22 @@ test('errorHandler in plugin should be separate from the external one', async t
   t.ok(fastify[kErrorHandler].func instanceof Function)
   t.same(fastify.errorHandler, fastify[kErrorHandler].func)
 })
+
+test('fastify instance should contains listeningOrigin property (with port and host)', async t => {
```

Comment:
Can you add one more testing for Unix sockets?

Reply:
just pushed a new commit with a new test case (not sure is exactly what you asked for)

Reply:
Passing a `path` property instead of `port` or `host` (or both), will create a Unix socket at the given `path`.
Once the server is closed, the "file" linked to that socket will be removed automatically.

```js
const start = async () => {
  const path = 'fastify.temp.sock';
  try {
    await app.listen({ path });
    app.log.info({ addresses: app.addresses() }) // returns ["fastify.temp.sock"]

    process.on('SIGINT', async () => {
      app.log.warn('closing server')
      await app.close();
    })
  } catch (error) {
    app.log.error(error);
    process.exit(1);
  }
};
```